src/orders/views/OrderList/OrderList.tsx

```diff
@@ -71,6 +71,12 @@ export const OrderList: React.FC<OrderListProps> = ({ params }) => {
 
   const { channel, availableChannels } = useAppChannel();
 
+  const channelOpts =
+    availableChannels?.map(channel => ({
```

Comment:
I can't really find any other uses of this function somewhere else than in this component (in `getFilterOpts`). Could you tell me where the code repeats?

Reply:
In the whole app, rather than this component. If you look for this object, you should be able to find it easily
```
({
   label: channel.name,
   value: channel.id
}))